react-hooks/intro-to-react-hooks/src/exercises/exercise2/solutions/part1Solution.js

```diff
@@ -15,15 +15,15 @@ const NumberTrivia = (props) => {
   );
   // 📗 Solution below 📗
   // We have also provided the second parameter to
-  // useEffect, an array of dependencies. We only
+  // useEffect, an array of values. We only
```

Comment:
I think we want to keep the word dependencies in here. The dependency array is a specific term used in relation to React Hooks ([see here](https://overreacted.io/a-complete-guide-to-useeffect/#teaching-react-to-diff-your-effects)) and I think maintaining that wording is helpful for new folks to this concept.

Reply:
Great! I changed it at first because for me at least it makes me relate it more with node dependencies, but I do agree on keeping it as dependencies to match docs